src/assets/scripts/client/utilities/unitConverters.js

```diff
@@ -159,6 +159,17 @@ export const kn_ms = (kn = 0) => {
     return kn * UNIT_CONVERSION_CONSTANTS.KN_MS;
 };
 
+/**
+ * nautical miles to feet
+ *
+ * @function nm_ft
+ * @param nm {number}
+ * @return {number}
+ */
+export const nm_ft = (nm) => {
```

Comment:
this should just be a normal function. Making it an arrow function causes some unnecessary overhead which is very insignificant here, but has been very significant elsewhere, so we've stuck to the pattern of using them only when specifically necessary, and instead using `function` and `.bind()` as appropriate for the added efficiency.
```
function nm_ft(nm) {
```

Reply:
Basically, I agree with you but the rest of the file uses the "export const"-syntax, so I would like to stick with that syntax or change the whole file to the function-syntax. Up to you.

Reply:
I don't have a problem with that.  I agree, it probably makes more sense to change all these at once.